src/view/options_menu.py

```diff
@@ -1,17 +1,129 @@
-from typing import List, Optional, Any, Dict
+from typing import List, Optional, Dict, Tuple
+from collections import OrderedDict
+
+import pygame
 
 from pygame.event import EventType
 from pygame.surface import Surface
+from pygame.rect import Rect
 
-from view.action import Action
+from view.action import Action, ActionType
 from view.view import View
+from view import colors
 
 
 class OptionsMenu(View):
-    def __init__(
-        self, available: Any, output_dataset: Dict[str, Any], next: Optional[int] = None
-    ) -> None:
+    def __init__(self, tracks: Dict[str, Action]) -> None:
         super().__init__()
+        pygame.font.init()
+        self.font = pygame.font.SysFont("Verdana", 30)
+        self.main_font = pygame.font.SysFont("comicsansms", 60)
+        self.selected_item = 0
+        self._options = OrderedDict(tracks)
+        self.font_color = colors.WHITE
+        self.button_highlight = colors.GREEN
+        self.button_color = colors.LIGHTGRAY
+        self._background: Optional[Tuple[Surface, Tuple[int, int]]] = None
+        self.background_image: Optional[Surface] = None
+        self.divider = 0.4
+        self.selected_action: Optional[Action] = None
+        self.img = pygame.image.load("resources/graphics/track1.jpg")
+        self.logo_image = pygame.image.load("resources/graphics/logo.png")
```

Comment:
Most of these fields are hardcoded (doesn't depend on constructor arguments) and never change. Consider using class constants.